Documentation/Introduction/Index.rst

```diff
@@ -66,12 +66,10 @@ From there you can hit the "Edit me on GitHub" button in the top right corner
 and submit a pull request via GitHub. Alternatively you can just file an issue
 using the bug tracker: https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-Editors/issues.
 
-To support the translation to other languages see
-`http://wiki.typo3.org/TYPO3_Tutorial_for_Editors
-<http://wiki.typo3.org/TYPO3_Tutorial_for_Editors>`_
+To support the translation to other languages see https://wiki.typo3.org/TYPO3_Tutorial_for_Editors.
```

Comment:
The wiki doesn't exits anymore. Should link to https://docs.typo3.org/m/typo3/tutorial-editors/7.6/en-us/ instead

Reply:
Yeah, there are still tons of wiki links in the documentation which all redirect to docs.typo3.org. They should be treated in a separate run with having a list at hand of which url to redirect where.